config/stage_2_crawler_tests/s2_5_raw_metadata_test.py

```diff
@@ -73,6 +73,7 @@ def test_validate_sort_metadata(self):
     @pytest.mark.mark8
     @pytest.mark.mark10
     @pytest.mark.stage_2_5_dataset_validation
+    @pytest.mark.skip(reason='because of nbsp spaces')
```

Comment:
remind me, please: who have you discussed the necessity of test skip with?

Reply:
we have discussed it on the lecture: Alexander Vladimirovich told me to skip this test, due to nbsp spaces on my source.

Reply:
OK, thank you